test/unit/auth.js

```diff
@@ -113,4 +115,224 @@ describe('DropboxAuth', () => {
       }
     });
   });
+
+  describe('clientSecret', () => {
+    it('can be set in the constructor', () => {
+      const dbx = new Dropbox({ clientSecret: 'foo' });
+      chai.assert.equal(dbx.auth.getClientSecret(), 'foo');
+    });
+
+    it('is undefined if not set in constructor', () => {
+      const dbx = new Dropbox();
+      chai.assert.equal(dbx.auth.getClientSecret(), undefined);
+    });
+
+    it('can be set after being instantiated', () => {
+      const dbx = new Dropbox();
+      dbx.auth.setClientSecret('foo');
+      chai.assert.equal(dbx.auth.getClientSecret(), 'foo');
+    });
+  });
+
+  describe('refreshToken', () => {
+    it('can be set in the constructor', () => {
+      const dbxAuth = new DropboxAuth({ refreshToken: 'foo' });
+      chai.assert.equal(dbxAuth.getRefreshToken(), 'foo');
+    });
+
+    it('is undefined if not set in constructor', () => {
+      const dbxAuth = new DropboxAuth();
+      chai.assert.equal(dbxAuth.getRefreshToken(), undefined);
+    });
+
+    it('can be set after being instantiated', () => {
+      const dbxAuth = new DropboxAuth();
+      dbxAuth.setRefreshToken('foo');
+      chai.assert.equal(dbxAuth.getRefreshToken(), 'foo');
+    });
+  });
+
+  describe('AccessTokenExpiresAt', () => {
```

Comment:
nit: We should keep naming consistent, others are lower case and this one is capitalized